apps/cli/src/node/docker-compose-espresso.yaml

```diff
@@ -15,9 +15,53 @@ services:
     validator:
         environment:
             CARTESI_FEATURE_INPUT_READER_ENABLED: false
+    espresso_reader_migrate:
+        image: golang:1.23
+        command:
+            - /bin/bash
+            - -c
+            - |
+                git clone --depth 1 https://github.com/cartesi/rollups-espresso-reader.git --branch v0.2.1-node-20250128
+                cd rollups-espresso-reader
+                make migrate
+        env_file:
+            - ${CARTESI_BIN_PATH}/node/default.env
+        depends_on:
+            database:
+                condition: service_healthy
+        environment:
+            DATABASE_URL: postgres://postgres:password@database:5432/espresso_reader
+
+    espresso_reader:
+        image: cartesi/sdk:devel
```

Comment:
```suggestion
        image: cartesi/sdk:0.12.0-alpha.5
```

Reply:
Already bumped to new sdk:0.12.0-alpha.6 that already containes espresso-dev-node with ;latest `patch5` tag.